### PR TITLE
Prepare Production, and make Staging private

### PR DESCRIPTION
**Pull request by dpordomingo**
After preparing Production environment https://github.com/src-d/code-annotation/pull/218 , some other changes must be done to:

- 196230c Convert Staging to a real testing env, making it accessible for [Applications members](https://github.com/orgs/src-d/teams/applications)
- 307509c Use production url as defined by https://github.com/src-d/code-annotation/issues/217#issuecomment-378615285

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -97,6 +97,6 @@ pipeline:
     <<: *helm_deploy
     prefix: PROD
     secrets: [ PROD_API_SERVER, PROD_KUBERNETES_TOKEN ]
-    values: ingress.globalStaticIpName=code-annotation-production,ingress.hostname=code-annotation.sourced.tech,image.tag=${DRONE_TAG},deployment.gaTrackingID=UA-109494282-2,secretName=code-annotation-drone-workaround
+    values: ingress.globalStaticIpName=code-annotation-production,ingress.hostname=code-annotation.srcd.run,image.tag=${DRONE_TAG},deployment.gaTrackingID=UA-109494282-2,secretName=code-annotation-drone-workaround
```
Comment (dpordomingo) [inline .drone.yml:100]: is this conf compatible @rporres ?
```
ingress.globalStaticIpName=code-annotation-production
ingress.hostname=code-annotation.srcd.run
```
I'm not sure if we should use a new `globalStaticIpName` suffixed as `-production` while we're having a `hostname` following the `staging` pattern.

Is this something I should care about?
Does it affect the place where you store the secrets and these configurations?
(most probably I'm mixing concepts, so please ping me when you have time to explain it to me irl :dancer: )

**Review by dpordomingo — COMMENTED** [review]
@rporres PTAL

**Review by bzz — APPROVED** [review]
LGTM. Let's wait for @rporres feedback as well.

Only App team members will be able to login to staging now.

I guess we just need to make sure that on production, same users as before would be able to log in, and see their previous answers.

**Comment by dpordomingo** [comment]
Since the staging DB will be copied to production, all workers will keep their previous work already done in Staging.
The Production ACL will be the same as currently in Staging: `restrictAccessGroup:org:src-d`, so nothing changed about it in Prod.

Does it answer your questions @bzz ?

File: .drone.yml (hunk 1 above)
Comment (rporres) [reply .drone.yml:100]: Please use `code-annotation-production.srcd.run` as hostname. The rest is ok

**Review by rporres — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (dpordomingo) [reply .drone.yml:100]: But that's the hostname defined by @mcuadros in the GH OAuth App as he wrote at https://github.com/src-d/code-annotation/issues/217#issuecomment-378615285

Should we ask him to change it?

Comment (rporres) [reply .drone.yml:100]: d'oh, ok. I don't like that name, but I guess it's late to change it. I will change it on my side...

**Review by rporres — APPROVED** [review]

**Pull request merged**